client/src/pages/Settings.js

```diff
@@ -39,39 +39,43 @@ const Settings = () => {
   const [isPlatformOpen, setPlatformOpen] = useState(false);
   const [isCompanyOpen, setCompanyOpen] = useState(false);
 
-  const handlePlatformClose = (event) => {
+  const handlePlatformClose = event => {
     setPlatformOpen(false);
-  }
+  };
 
-  const handlePlatformOpen = (event) => {
+  const handlePlatformOpen = event => {
     setPlatformOpen(true);
-  }
+  };
 
-  const handlePlatformChange = (event) => {
+  const handlePlatformChange = event => {
     setSelectedPlatforms(event.target.value);
-  }
+  };
 
-  const handleCompanyClose = (event) => {
+  const handleCompanyClose = event => {
     setCompanyOpen(false);
-  }
+  };
 
-  const handleCompanyOpen = (event) => {
+  const handleCompanyOpen = event => {
     setCompanyOpen(true);
-  }
+  };
 
-  const handleCompanyChange = (event) => {
+  const handleCompanyChange = event => {
     setSelectedCompanies(event.target.value);
-  }
+  };
 
-  const handleSearchSubmit = (event) => {
+  const handleSearchSubmit = event => {
     event.preventDefault();
-    console.log(event.target.searchfield.value, event.target.companyfield.value, event.target.platformfield.value);
+    console.log(
+      event.target.searchfield.value,
+      event.target.companyfield.value,
+      event.target.platformfield.value
+    );
     setSearch("");
-  }
+  };
 
-  const onSearchChange = (event) => {
+  const onSearchChange = event => {
     setSearch(event.target.value);
-  }
+  };
```

Comment:
Should probably undo these changes since I put the functions for the searchbar in the searchbar component on my pr so this would just add in unneeded functions